services/dedup/dedup.go

```diff
@@ -50,7 +50,7 @@ func (l loggerForBadger) Warningf(fmt string, args ...interface{}) {
 }
 
 func DefaultRudderPath() string {
-	badgerPathName := "/badgerdbv2"
+	badgerPathName := "/badgerdbv3"
 	tmpDirPath, err := misc.CreateTMPDIR()
 	if err != nil {
 		panic(err)
```

Comment:
Can't we return the error to the caller and handler it appropriately there without panicking?